src/client/src/App.js

```diff
@@ -58,10 +64,31 @@ class App extends Component {
     this.setState({[e.target.name]: e.target.value})
   }
 
+  getLinks = (bundle) => {
+    console.log('BUNDLE', bundle);
```

Comment:
Do we still need these console logs or can they be removed?

Reply:
I want to keep the logs for now because they're useful for debugging errors we get when encountering an object that isn't part of a bucket in `patientResourceCategories`

Reply:
Sounds good!